packages/client/src/forms/index.ts

```diff
@@ -468,6 +470,7 @@ export interface INumberFormField extends IFormFieldBase {
   type: typeof NUMBER
   step?: number
   max?: number
+  inputFieldWidth?: string
```

Comment:
I started trying to fix some of the typescript issues but have to continue in another PR.